sql/execution_pipeline_impl.cc

```diff
@@ -256,7 +270,7 @@ take()
     //cerr << "got row " << current[currentDone].rowName << " "
     //     << jsonEncodeStr(current[currentDone].columns) << endl;
 
-    result->values.emplace_back(current[currentDone].rowName.toSimpleName(),
+    result->values.emplace_back(current[currentDone].rowName,
```

Comment:
Thanks.  I wasn't sure how to store a path in an ExpressionValue.  Having a dedicated cell value type is better.